common/djangoapps/course_about/tests/test_data.py

```diff
@@ -0,0 +1,74 @@
+"""
+Tests specific to the Data Aggregation Layer of the Course About API.
+
+"""
+import unittest
+from django.test.utils import override_settings
+from django.conf import settings
+from xmodule.modulestore.tests.django_utils import (
+    ModuleStoreTestCase, mixed_store_config
+)
+from xmodule.modulestore.tests.factories import CourseFactory
+from student.tests.factories import UserFactory
+from course_about import data
+from course_about.errors import CourseNotFoundError
+from nose.tools import raises
+from xmodule.modulestore.django import modulestore
+from datetime import datetime
```

Comment:
By convention, we put standard library imports at the top.  See https://github.com/edx/edx-platform/wiki/Python-Guidelines#imports-order